admin-ui/plugins/auth-server/components/Clients/ClientBasicPanel.js

```diff
@@ -70,7 +88,39 @@ const ClientBasicPanel = ({
   function handleMouseDownClientSecret(event) {
     event.preventDefault()
   }
-  
+
+  useEffect(() => {
+    const scopeInums = [];
+    if (client.inum) {
+      let userAction = {}
+      for (const scope of client.scopes) {
+        scopeInums.push(getClientScopeByInum(scope))
+      }
+      userAction["pattern"] = scopeInums.join(",")
+      userAction["limit"] = PER_PAGE_SCOPES
+      dispatch(getClientScopes(userAction))
+    }
+  }, [])
+
+  const handlePagination = async (event, shownResults) => {
```

Comment:
Hi @jv18creator , as of now you are fetching record with increasing limit, but in the API's there is a limit of 200 records can be fetched in a single call. So instead of increasing the limit you need to implement pagination api there.

<img width="1144" alt="Screenshot 2023-03-27 at 11 17 25 PM" src="https://user-images.githubusercontent.com/32476704/228026225-cedb8c7b-c3db-4df0-a16e-a8d4b94b33e6.png">
this is the error you will get once the records are more than 200

kindly use api like this : `http://localhost:8080/jans-config-api/api/v1/scopes?limit=10&startIndex=10&withAssociatedClients=true`

in this limit is 10 and start index indicated where's from we need to fetch the records. For more please check the scopes page pagination it's already implemented there :)